src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1

```diff
@@ -584,24 +584,25 @@ function Get-Location
 {
     param([string]$providerNamespace, [string]$resourceType, [string]$preferredLocation)
     $provider = Get-AzureRmResourceProvider -ProviderNamespace $providerNamespace
-    $resourceTypes = $null
```

Comment:
Common changes are not needed here. Should be done in common repo.

Reply:
Thee changes are already present in common, I just wanted to keep the files in both the repos on the same page...

Reply:
@praries880 These files will be deleted soon. I have to resolve https://github.com/Azure/azure-powershell/issues/6946 and then I'm going to delete these files. I'll hopefully have a PR next week.

Reply:
Then it does not hurt to leave the changes as is does it? The changes are already there in the new repo.. n this file is not used anyhow.

Reply:
After we created the common repo, we said we wouldn't take anymore changes here. The rationale being that it would only cause confusion to update anything in these directories, since these files are unused. If you just copy/paste the current version of the file in the repo to your fork, you'll effectively 'revert' it.